### ISSUE TITLE
Unable to upload anything when clicking Upload ID button on Tamara

### ISSUE THREAD

**Issue by AsimNGI** [Shopify/checkout-sheet-kit-android#115]
### What area is the issue related to?

Checkout Sheet Kit

### What version of the package are you using?

3.0.2

### Steps to Reproduce

1. Open checkout
2. Select option with any installment plan, in my case it's Tamara.
3. Click on Pay now.
4. Otp Verification
5. It will ask to Upload ID of front & back but button is not doing anything.

### Expected Behavior

There can be two ideal cases:

1. If Camera permission is not given it should ask for Camera permission and then opens camera to capture picture.
2. If camera permission is given, it should directly opens camera.

### Actual Behavior

By clicking on Upload ID nothing happened. 

### Screenshots/Videos/Log output

https://github.com/user-attachments/assets/e2e0f904-5029-46f7-a497-d2d6aab8ada8

### Storefront domain

https://revibe.me/

**Comment by AsimNGI** [comment]
Yup, already gave permission for the microphone and camera. I've also
attached the screenshot of app permissions.

**Comment by AsimNGI** [comment]
Yes, tried this solution already but still got the same issue.

**Comment by AsimNGI** [comment]
We can provide the keys or any support required from Tamara to expedite the
process.

**Comment by AsimNGI** [comment]
@kiftio can you please give us any update regarding the issue ?

**Comment by kiftio** [comment]
Ah unfortunately, the first step is to verify a mobile number, which must be a UAE number. So, I'm still a bit stuck I'm afraid

**Comment by AsimNGI** [comment]
Any update?

**Comment by kiftio** [comment]
I can't test Tamara without a UAE number unfortunately. So no, no update. 

Can you see if there are any logs when the page is loaded / button clicked?

**Comment by AsimNGI** [comment]
The logs are not showing anything when clicking on button.
Can we align a meeting? So we can give do testing on UAE number ?

**Comment by AsimNGI** [comment]
@kiftio  Waiting for your response to expedite the process.

**Comment by allyshiasewdat** [comment]
Hi @AsimNGI - This is important to us, and we will have the bandwidth to investigate this further at the end of August at the earliest.

In the meanwhile, were you able to test whether this works in a standard web view, without the Checkout Sheet Kit?

**Comment by kiftio** [comment]
@AsimNGI Few more questions to help us double check things:

- Are you testing via the emulator or on a real device?
- Which Android OS version are you using?
- Can you try in a different device (different model, different OS) and report back if you see the same issue?

Thanks

**Comment by AsimNGI** [comment]
I'm testing in a real device and it's not working in all android models of
all brands and tested in all OS versions. Still same issue is coming.

**Comment by AsimNGI** [comment]
@kiftio  Any update ? We're loosing 20% of our customers due to this bug. Can we schedule a meeting at your earliest? and we can provide you our UAE number to test

**Comment by LouisMVA** [comment]
@AsimNGI - We've been working through your merchant success manager to setup a call so we can debug further. There should have been an email sent to your team last week to coordinate a time to meet. Please follow up there and we'll be happy to provide further guidance.

**Comment by AsimNGI** [comment]
Hi @LouisMVA! Where have you sent the email. We are unable to find any and can you please keep my email in loop as well?

**Comment by LouisMVA** [comment]
Mind sharing your email with me? I can make sure you get copied

**Comment by AsimNGI** [comment]
Sure! My email is ***@***.***

**Comment by AsimNGI** [comment]
Sure, my email is asim.zubair@nextgeni.com.

**Comment by AsimNGI** [comment]
Hi @LouisMVA ! We had multiple connects with @kiftio regarding this issue. However, we are stuck in only 1 edge case which is regarding to the camera. We tested in multiple devices and here are the results:
Image Upload:

When we pick an image using the image picker, everything works fine on all devices we've tested so far.

Camera:

1. Initially, the camera worked fine on two devices: Google Pixel (Android 11) and Samsung (Android 11).

2. However, when we retested the checkout flow on the same Samsung device, we were unable to place the photo on Tamara's checkout WebView after taking and selecting the picture. (Video 1)

3. We encountered the same issue (issue #2) when testing on an Infinix phone (Android 13). (Video 1)

4. On a Xiaomi phone (Android 14), after taking a picture and selecting the photo, the screen blacked out, and we had to close the app. (No video, happened all of sudden)

5. We retested on the same Xiaomi device, and after selecting the photo by taking a picture, the checkout flow was dismissed. (Video 2).

[https://drive.google.com/file/d/1P6p669D1C7rRuqWtgF9LPxJdnhK13is_/view?usp=drive_link](Video 1)
[https://drive.google.com/file/d/1P9whb_ym0AIGKrVyg4Tf9jZniqBA4p_D/view?usp=drive_link](Video2)
